devops/windows/base-docker-wsl/helper/log/logs-zookeeper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -u
+
+source 0-conf.env
+docker service logs --raw ${STACK_NAME}_zookeeper
```

Comment:
Aint we using kraft instead...Never saw any zookeeper configuration on  the docker stack?